spec/support/good_job_test_helper.rb

```diff
@@ -0,0 +1,33 @@
+require 'fileutils'
+
+module GoodJobTestHelper
+  def create_dummy_app
+    FileUtils.cd(tmp_path) do
+      `rails new dummy --skip-action-mailer --skip-action-mailbox --skip-action-cable --skip-sprockets --skip-listen --skip-javascript --skip-turbolinks --skip-system-test --skip-test-unit --skip-bootsnap --skip-spring`
```

Comment:
I'd like to avoid using "dummy" because of inclusive language. `example_app` instead?

```suggestion
      `rails new example_app --skip-action-mailer --skip-action-mailbox --skip-action-cable --skip-sprockets --skip-listen --skip-javascript --skip-turbolinks --skip-system-test --skip-test-unit --skip-bootsnap --skip-spring`
```